src/sdk/base/src/main/java/owt/base/LocalStream.java

```diff
@@ -77,4 +77,8 @@ public void dispose() {
         mediaStream.dispose();
         mediaStream = null;
     }
+
+    public boolean disposed() {
```

Comment:
It would be better to have descriptions for public functions.

Reply:
Hello jianjunz, I've added a description for the function.

Reply:
Thanks.